homeassistant/components/advantage_air/climate.py

```diff
@@ -171,32 +174,32 @@ class AdvantageAirZone(AdvantageAirClimateEntity):
     _attr_hvac_modes = ZONE_HVAC_MODES
     _attr_supported_features = SUPPORT_TARGET_TEMPERATURE
 
-    def __init__(self, instance, ac_key, zone_key):
+    def __init__(self, instance: advantage_air, ac_key: int, zone_key: str) -> None:
```

Comment:
ac_key is a string
https://github.com/home-assistant/core/blob/dev/tests/components/advantage_air/fixtures/getSystemData.json#L3